tests/test_commandline

```diff
@@ -174,4 +174,66 @@ fi
 
 echo "Test 3 passed"
 
+# Test 4: --tpmstate backend=file parameters test
+TPMDIR=`mktemp -d`
+PID_FILE=$TPMDIR/${SWTPM}.pid
+FILEMODE=641
+
+$SWTPM_EXE socket \
+	-p $PORT \
+	--tpmstate backend=file,backend_uri=$TPMDIR,mode=$FILEMODE \
+	--pid file=$PID_FILE \
+	--flags not-need-init \
+	${SWTPM_TEST_SECCOMP_OPT} &
+PID=$!
+
+if wait_port_open $PORT $PID 4; then
+	echo "Test 4 failed: TPM did not open port $PORT"
+	exit 1
+fi
+
+kill_quiet -0 $PID
+if [ $? -ne 0 ]; then
+	echo "Test 4 failed: TPM process not running"
+	exit 1
+fi
+
+if wait_for_file $PID_FILE 3; then
+	echo "Error: ${SWTPM_INTERFACE} TPM did not write pidfile."
+	exit 1
+fi
+
+validate_pidfile $PID $PID_FILE
+
+${SWTPM_BIOS} &>/dev/null
+if [ $? -ne 0 ]; then
+	echo "Test 4 failed: tpm_bios did not work"
```

Comment:
tpm_bios -> swtpm_bios

Reply:
or rather `${SWTPM_BIOS}`

Reply:
Thanks, I'll fix that and also "Test 1" as well.